test/framework/alertmanager.go

```diff
@@ -215,11 +215,14 @@ func amImage(version string) string {
 }
 
 func (f *Framework) WaitForAlertmanagerInitialized(ns, name string, amountPeers int, forceEnableClusterMode bool) error {
+
```

Comment:
Nitpick: but we tend to not do a new line after function decleration, so this is not really needed.

Reply:
Let me fix it quickly